src/fw/drivers/task_watchdog.c

```diff
@@ -36,6 +36,7 @@
 #define STM32F4_COMPATIBLE
```

Comment:
Rather than throw a bunch of board defines in here, can we just make a stub/disabled config flag?

Reply:
let me add a task watchdog stub driver